docs/src/domains/cdn/fastly.md

```diff
@@ -1,55 +1,55 @@
 ---
-title: "Fastly"
+title: "Set up your Fastly CDN"
+sidebarTitle: "Fastly setup"
 weight: 1
-description: In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+description: Learn how to configure your Fastly CDN.
+aliases:
+ - "/golive/steps/fastly.html"
```

Comment:
:clipboard: We shouldn't use aliases: https://github.com/platformsh/platformsh-docs/blob/main/CONTRIBUTING.md#redirect-pages